operator/deploy/psp.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kiali-operator
+  labels:
+    app: kiali-operator
+    version: ${OPERATOR_VERSION_LABEL}
+spec:
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - downwardAPI
```

Comment:
As per https://kubernetes.io/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems they recommend a minimum set of the following:

```
- configMap
- downwardAPI
- emptyDir
- persistentVolumeClaim
- secret
- projected
```

Of that list, it looks like you are missing `persistentVolumeClaim` and `projected`. I think you should add those unless you have a reason not to.